### PR TITLE
Rename variable.

### LINKED ISSUE #108: Change variable name in template
This issue is to rename the variable `line` in `regression.py` so it does not cause the unit tests to fail. This variable is written in the `mos` file. This causes a name clash for instances of models which have the name `line`. This happens for instance in the model  `Buildings.Electrical.DC.Lines.Examples.DCLinesLinearized`.

### PR DESCRIPTION
**Pull request by tsnouidui**
This fixes #108.

### CODE REVIEW

File: buildingspy/development/regressiontest.py [hunk 1]
```diff
@@ -1968,7 +1968,7 @@ def _print_end_of_json(isLastItem, fileHandle, logFileName):
   iLin=1;
   endOfFile=false;
   while (not endOfFile) loop
-    (line, endOfFile)=Modelica.Utilities.Streams.readLine("{modelName}.dslog.log", iLin);
+    (_line, endOfFile)=Modelica.Utilities.Streams.readLine("{modelName}.dslog.log", iLin);
     iLin=iLin+1;
     iSuc=iSuc+Modelica.Utilities.Strings.count(line, "Integration terminated successfully");
```
Comment (mwetter) [inline buildingspy/development/regressiontest.py]: you also need to change this line

File: buildingspy/development/regressiontest.py [hunk 2]
```diff
@@ -2009,7 +2009,7 @@ def _print_end_of_json(isLastItem, fileHandle, logFileName):
   iLin=1;
   endOfFile=false;
   while (not endOfFile) loop
-    (line, endOfFile)=Modelica.Utilities.Streams.readLine("{modelName}.dslog.log", iLin);
+    (_line, endOfFile)=Modelica.Utilities.Streams.readLine("{modelName}.dslog.log", iLin);
     iLin=iLin+1;
     iSuc=iSuc+Modelica.Utilities.Strings.count(line, "Created {FMUName}");
```
Comment (mwetter) [inline buildingspy/development/regressiontest.py]: and this line.

**Comment by tsnouidui** [comment]
@mwetter: This is fixed now. Thanks!

**Comment by tsnouidui** [comment]
@mwetter: Could you please merge this pull request?

**Pull request merged**